index.html

```diff
@@ -6,8 +6,28 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="styles.css">
     <title>Portfolio Skeleton</title>
+    <script src="https://kit.fontawesome.com/5211805414.js" crossorigin="anonymous"></script>
 </head>
 <body>
-    
+    <header>
+        <nav>
+            <ul>
+                <li><a href="https://es.lovepik.com/image-450040433/programming-vector.html" target="_blank" rel="noopener"><img src="./Normal Button/Tertiary/Icons/logo3.png" alt="My-Logo"></a></li>
```

Comment:
- [ ] Excellent work implementing your semantic `a` tag to your logo 👌. However, when I click on the logo an external web is displayed, please kindly make sure that when the logo is clicked, no external webs are opened since this is your personal portfolio 🤓